Userland/Libraries/LibWeb/CSS/Properties.json

```diff
@@ -72,11 +72,11 @@
     "inherited": false
   },
   "border-bottom-left-radius": {
-    "initial": 0,
+    "initial": "0",
```

Comment:
Perhaps explain why this change is needed (or useful) in the commit description?

Reply:
:+1: